pycsw/core/repository.py

```diff
@@ -816,6 +817,7 @@ def setup(database, table, create_sfsql_tables=True, postgis_geometry_column='wk
         Column('publisher', Text, index=True),
         Column('contributor', Text, index=True),
         Column('organization', Text, index=True),
+        Column('funder', Text, index=True),
```

Comment:
The constitutes a model change (and DB migration for existing installations).  Just want to confirm that `funder` is a valuable queryable and column to have in the database model.

Reply:
@tomkralidis We are under some pressure to recognise sources of funding in all our metadata. This helps us justify to our funders that their money does have valuable and tangible outcomes.   But it is your call in the end.